plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -2183,4 +2183,27 @@ public void testAggregationPushdownWithArrays()
                         "(56, VARCHAR 'string_8400', BIGINT '1')," +
                         "(1000, VARCHAR 'string1_8401', BIGINT '1')");
     }
+
+    @Test
+    public void testHavingClause()
+    {
+        assertThat(query("SELECT stringcol, \"sum(longcol)\"" +
```

Comment:
This assertion should belong to "Add having support to Pinot passthrough queries" commit in strict.